systems/billing/api-gateway/pkg/client/invoice.go

```diff
@@ -0,0 +1,120 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright (c) 2023-present, Ukama Inc.
+ */
+
+package client
+
+import (
+	"context"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	log "github.com/sirupsen/logrus"
+	pb "github.com/ukama/ukama/systems/billing/invoice/pb/gen"
+)
+
+type Invoice interface {
+	Add(rawInvoice string) (*pb.AddResponse, error)
+	Get(invoiceId string, asPDF bool) (*pb.GetResponse, error)
+	List(invoiceeId, invoiceeType, networkId string,
```

Comment:
Why do we write `invoiceeId` instead of `invoiceId`

Reply:
we need both.